client/Wayland/wlf_input.c

```diff
@@ -441,7 +444,11 @@ BOOL wlf_handle_touch_up(freerdp* instance, const UwacTouchUp* ev)
 		return FALSE;
 
 	WINPR_ASSERT(rdpei->TouchEnd);
+	int contactId;
```

Comment:
Coding style. Even if a another if defined is required this should be at the beginning of the function.